script/fs-bench/work/Dockerfile

```diff
@@ -0,0 +1,43 @@
+#   Copyright The containerd Authors.
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+FROM golang:1.13
+
+# basic tools
+COPY ./tools /tmp/tools
+
+RUN apt-get update -y && \
+    apt-get install --no-install-recommends -y libbtrfs-dev libseccomp-dev fuse python \
+        apt-transport-https software-properties-common && \
+    curl -Lo gnuplot-5.2.8.tar.gz \
+    https://sourceforge.net/projects/gnuplot/files/gnuplot/5.2.8/gnuplot-5.2.8.tar.gz/download && \
+    tar xf gnuplot-5.2.8.tar.gz && \
+    cd gnuplot-5.2.8 && \
+    ./configure && make && make install && \
+    cd /tmp/tools && \
+    GO111MODULE=on go build -o "/usr/local/bin/process" "./process/main.go" && \
+    GO111MODULE=on go build -o "/usr/local/bin/scrape" "./scrape/main.go"
+
+# runtime dependencies
+RUN git clone https://github.com/opencontainers/runc \
+          $GOPATH/src/github.com/opencontainers/runc && \
+    cd $GOPATH/src/github.com/opencontainers/runc && \
+    git checkout d736ef14f0288d6993a1845745d6756cfc9ddd5a && \
+    GO111MODULE=off make -j2 BUILDTAGS='seccomp apparmor' && \
+    GO111MODULE=off make install && \
+    git clone https://github.com/containerd/containerd \
+          $GOPATH/src/github.com/containerd/containerd && \
+    cd $GOPATH/src/github.com/containerd/containerd && \
+    git checkout 990076b731ec9446437972b41176a6b0f3b7bcbf && \
+    GO111MODULE=off make -j2 && GO111MODULE=off make install
```

Comment:
Recently we introduced [a common base image](https://github.com/containerd/stargz-snapshotter/blob/master/Dockerfile) among test codes for easier version management of containerd & runc in this project. Can we use the base image for this? Can use `snapshotter-base` build target which includes runc + containerd + containerd-stargz-grpc(built from the repo) so we won't need to build the snapshotter binary inside the testing container during runtime.
Please also see the script in the [integration test](https://github.com/containerd/stargz-snapshotter/blob/875ec333403a885f5b6e5b64c94ec4dc713e0596/script/integration/test.sh#L32-L79).